source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -1,50 +1,91 @@
-title: Create an AWS IAM User
-ref: create-an-aws-iam-user
+title: Create a GCP Service Account
+ref: create-a-gcp-service-account
 content: |
-  Create a new programmatic IAM user in the AWS management console.
-  CSFLE-enabled clients authenticate with AWS KMS using the IAM user to
-  encrypt and decrypt the remote master key. The IAM user must be granted
-  full ``List`` and ``Read`` permissions for the KMS service.
 
-  .. note:: Client IAM User Credentials
+  Create a service account for your client application by following the
+  official Google documentation on
+  `Creating and managing a service account <https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating>`__.
+
+  After you create the service account, locate and record the following
+  authentication settings for use in a later step:
+
+  - **email**
+  - **privateKey** (this value is only returned when you create the account)
+  - **endpoint**
```

Comment:
Suggestion: These might work better as substeps, since you need to do them in order. Something like:

a. Create a service account for your client application. See the Google documentation for instructions.
b. Locate and record the following authentication settings:

Reply:
Thanks, this makes sense. I'm going to move this suggestion to a new issue since it applies to content in multiple places.

Reply:
https://jira.mongodb.org/browse/DOCSP-13089?filter=35806